README.md

```diff
@@ -139,15 +139,16 @@ Assuming the name of the created Sandbox is named `foo`, the following resources
 
 ### Namespace (sandbox-foo)
 
-### ClusterRole (sandbox-foo-deleter)
+### ClusterRole (sandbox-foo-admin)
 
 |Verbs|API Groups|Resources|ResourceNames|
 |---|---|---|---|
 |delete|operators.plex.dev|sandboxes|sandbox-foo|
+|patch|operators.plex.dev|sandboxes|sandbox-foo|
```

Comment:
nit: At a minimum I think we should include the `update` verb here as well (on the same row). Though to tell the truth, I can't think of a reason not to just allow all verbs on this resource.

Reply:
True...maybe I'll update to give allow all verbs for that clusterrole.  I'll commit the updates and see what you think.